spec/widgets/category/category-widget-model.spec.js

```diff
@@ -32,6 +33,23 @@ describe('widgets/category/category-widget-model', function () {
     });
   });
 
+  describe('search state', function () {
```

Comment:
I looked at the code that we are testing, and I'm now wandering If that code actually belongs to deep-insights.js:

There's this binding:

```
  this.listenTo(this.dataviewModel, 'change:searchData', this._onDataviewChangeSearchData);
```

And this callback:

```
 _onDataviewChangeSearchData: function () {
    // Update selected state for each search result item based on locked categories
    this.dataviewModel.getSearchResult().each(function (m) {
      var selected = this.dataviewModel.filter.isAccepted(m.get('name'));
      m.set('selected', selected);
    }, this);
  },
```

It's basically doing stuff on the dataview (not using anything from the widget model as far as I can tell), so perhaps it should be moved to CartoDB.js?

cc: @viddo